app/react-native/src/manager/provider.js

```diff
@@ -42,16 +47,6 @@ export default class ReactProvider extends Provider {
     const renderPreview = addons.getPreview();
 
     const innerPreview = renderPreview ? renderPreview(kind, story) : null;
-
-    if (this.options.manualId) {
```

Comment:
@Gongreg Was this just for debugging? Any implications to removing it?

Reply:
@shilman Kinda. That is what I used before to connect phone with browser.